Libraries/oneMKL/matrix_mul_mkl/matrix_mul_mkl_bf16.cpp

```diff
@@ -0,0 +1,83 @@
+// Standard SYCL header
+#include <CL/sycl.hpp>
+// STL classes
+#include <exception>
+#include <iostream>
+// Declarations for Intel oneAPI Math Kernel Library SYCL/DPC++ APIs
+#include "oneapi/mkl.hpp"
+int main(int argc, char *argv[]) {
+
+	// User obtains data here for A, B, C matrices, along with setting m, n, k, ldA, ldB, ldC.
+    // For this example, A, B and C should be initially stored in a std::vector,
+    //   or a similar container having data() and size() member functions.
+
+using namespace oneapi::mkl;
+int64_t m = 10, n = 6, k = 8, ldA = 12, ldB = 8, ldC = 10;
+int64_t sizea = ldA * k, sizeb = ldB * n, sizec = ldC * n;
+double alpha = 1.0, beta = 0.0;
+// Allocate matrices
+std::vector<bfloat16> A(sizea);
+std::vector<bfloat16> B(sizeb);
+std::vector<bfloat16> C(sizec);
+// Initialize matrices […]
+std::fill(A.begin(), A.end(), 0);
+std::fill(B.begin(), B.end(), 0);
+std::fill(C.begin(), C.end(), 0);

```

Comment:
Please clean up the indenting/formatting here (formatting should match matrix_mul_mkl.cpp).